official/mnist/mnist_eager.py

```diff
@@ -31,10 +31,11 @@
 import sys
 import time
 
-import tensorflow as tf
-import tensorflow.contrib.eager as tfe
+import tensorflow as tf  # pylint: disable=g-bad-import-order
+import tensorflow.contrib.eager as tfe  # pylint: disable=g-bad-import-order
+
+from official.mnist import dataset as mnist_dataset
 from official.mnist import mnist
-from official.mnist import dataset
 
```

Comment:
I think glint will give u a snippet to paste for incorrect order, should we do that instead of put a lint suppression here?

Reply:
The problem is TF versus official; alphabetically TF should come after official, and to Google, their both the same, but for us, TF is third party.